arm-network/2017-06-01/swagger/loadBalancer.json

```diff
@@ -557,10 +558,11 @@
         },
         "protocol": {
           "type": "string",
-          "description": "The transport protocol for the endpoint. Possible values are: 'Udp' or 'Tcp'",
+          "description": "The transport protocol for the endpoint. Possible values are: 'Udp,' 'Tcp,' or 'All.'",
```

Comment:
I don't understand the question - is this the same thing you were asking with regards to `Udp`?